README.md

```diff
@@ -1 +1,34 @@
 # Bookstore
+
```

Comment:
- [OPTIONAL] Please it will be nice to add a short description about your project as this will enable users to quickly have an idea of what your project is about.